jupyter-lib/kotlin-jupyter-api/src/main/kotlin/org/jetbrains/kotlin/jupyter/api/results.kt

```diff
@@ -0,0 +1,55 @@
+package org.jetbrains.kotlin.jupyter.api
+
+import com.beust.klaxon.JsonObject
```

Comment:
Why klaxon and not Serialization runtime? Serialization runtime is small and more widely used meaning that  you won't have to convert between different json representations.

Reply:
Migrated to kotlinx-serialization